src/ChangeLog:

2005-05-27  John W. Eaton  <jwe@octave.org>

	* ls-mat5.cc (save_mat5_binary_element): Extract complex N-d
	array, not matrix.


Index: src/ls-mat5.cc
===================================================================
RCS file: /usr/local/cvsroot/octave/src/ls-mat5.cc,v
retrieving revision 1.8.2.5
diff -u -r1.8.2.5 ls-mat5.cc
--- src/ls-mat5.cc	5 May 2005 03:53:38 -0000	1.8.2.5
+++ src/ls-mat5.cc	27 May 2005 14:14:32 -0000
@@ -1180,7 +1180,7 @@
     }
   else if (tc.is_complex_scalar () || tc.is_complex_matrix ()) 
     {
-      ComplexNDArray m_cmplx = tc.complex_matrix_value ();
+      ComplexNDArray m_cmplx = tc.complex_array_value ();
 
       write_mat5_array (os, ::real (m_cmplx), save_as_floats);
       write_mat5_array (os, ::imag (m_cmplx), save_as_floats);
